src/community/geofence-server/src/main/resources/applicationContext.xml

```diff
@@ -56,7 +56,8 @@
         <property name="locations">
             <list>
                 <value>classpath*:geofence-datasource-ovr.properties</value>
-                <value>${GEOSERVER_DATA_DIR}/geofence/geofence-datasource-ovr.properties</value> <!-- optional -->
+                <!-- This location is relative to the datadir -->
+                <value>file:geofence/geofence-datasource-ovr.properties</value> <!-- optional -->
```

Comment:
Yes, and in this case it might not even be a file, but a resource in database.
What is the alternative? Can we create our own scheme resourcestore: or something?